purchases/src/main/java/com/revenuecat/purchases/interfaces/ReceivePurchaserInfoListener.kt

```diff
@@ -0,0 +1,33 @@
+//  Purchases
+//
+//  Copyright © 2019 RevenueCat, Inc. All rights reserved.
+//
+package com.revenuecat.purchases.interfaces
+
+import com.revenuecat.purchases.CustomerInfo
+import com.revenuecat.purchases.PurchaserInfo
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.toPurchaserInfo
+
+/**
+ * Interface to be implemented when making calls that return a [PurchaserInfo]
+ */
+@Suppress("unused")
+@Deprecated(
+    "Renamed to ReceiveCustomerInfoListener",
+    level = DeprecationLevel.WARNING,
```

Comment:
Made this just warning becase they are normally used in functions that are deprecated with ERROR, so it doesn't really matter